.github/workflows/create-release-tag.yml

```diff
@@ -88,21 +89,44 @@ jobs:
             exit 1
           fi
 
-      - name: Create and push tag
+      # GPG setup for signed tags
```

Comment:
we need signed commits. this is copied from our existing npm release flow [here](https://github.com/recallnet/js-recall/blob/bacf524854fc0f75ae617915cecb3e169426212c/.github/workflows/release-staging.yml#L27)